packages/neuron-wallet/src/utils/multisig.ts

```diff
@@ -27,6 +32,27 @@ export const getMultisigStatus = (multisigConfig: MultisigConfigModel, signature
   return SignStatus.Signed
 }
 
+export const parseMultisigTxJsonFromCkbCli = (tx: OfflineSignJSON): Transaction => {
+  const { multisig_configs, transaction } = tx
+  // @ts-expect-error
+  const txObj = Transaction.fromObject(deepCamelizeKeys(transaction))
```

Comment:
`Transaction.fromObject(deepCamelizeKeys(transaction) as Parameters<Transaction['fromObject']>[0])` or `as any` is more suitable than `ts-expect-error` annotation here